frontend/nginx/conf.d/default.conf

```diff
@@ -0,0 +1,11 @@
+server {
```

Comment:
You can probably use one of the sidecar NGINX, like this https://github.com/aodn/aodn-stac/blob/main/docker-compose.yml#L48

Reply:
Well, it's a good idea, except you have your SPA building and being served from the nginx app, so it's a difference use case. This works as a proxy, though :-)